js/app2.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+class Book {
+  #list
+
+  constructor() {
+    const localList = JSON.parse(localStorage.getItem('stuff'));
+    if (localList !== null) {
+      this.#list = localList.arr;
+    }
+  }
+
+  #g(j) {
```

Comment:
* I suggest you use proper names for variables. Variables are so important to the code that they deserve a good name that accurately describes their purpose.